src/test/java/gov/healthit/chpl/aqa/asserts/ViewProductPageAsserts.java

```diff
@@ -0,0 +1,85 @@
+package gov.healthit.chpl.aqa.asserts;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.fail;
+
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import cucumber.api.java.en.Then;
+import gov.healthit.chpl.aqa.pageObjects.ViewProductPage;
+import gov.healthit.chpl.aqa.stepDefinitions.Base;
+
+/**
+ * Class ViewProductPageAsserts definition.
+ */
+public class ViewProductPageAsserts extends Base {
+
+    /**
+     * Assert that product edit link exists on view version page.
+     * @param productId is the product id passed in page objects
+     */
+    @Then("^I see product edit link for product \"([^\"]*)\"$")
+    public void iSeeProductEditLink(final String productId) {
+        try {
+            ViewProductPage.editProductLink(getDriver(), productId).isDisplayed();
+            assertTrue(true);
+        } catch (NoSuchElementException e) {
+            fail("Unable to find product edit link");
+        }
+    }
+
+    /**
+     * Assert that product split link exists on view version page.
+     * @param productId is the product id passed in page objects
+     */
+    @Then("^I see product split link for product \"([^\"]*)\"$")
+    public void iSeeProductSplitLink(final String productId) {
+        try {
+            ViewProductPage.splitProductLink(getDriver(), productId).isDisplayed();
+            assertTrue(true);
+        } catch (NoSuchElementException e) {
+            fail("Unable to find product split link");
+        }
+    }
+
+    /**
+     * Assert that product merge link exists on view version page.
+     * @param productId is the product id passed in page objects
+     */
+    @Then("^I see product merge link for product \"([^\"]*)\"$")
+    public void iSeeProductMergeLink(final String productId) {
+        try {
+        assertTrue(ViewProductPage.mergeProductLink(getDriver(), productId).isDisplayed());
```

Comment:
Your indentation here is off. I know it's a small thing, but checkstyle should catch these too. Recommend running a checkstyle report and verifying there aren't any real issues